.dockerignore

```diff
@@ -1,7 +1,7 @@
-.git/*
-conf/*
-packager/*
-scripts/*
+.git
+conf
+packager
+scripts
```

Comment:
I see people are using `.git` in stead of `.git/*` but when I do `docker build .`, it still sends whole `.git` directory to container... maybe it just a bug on my machine.. :joy: